### PR TITLE
Add docs for time / date styling

### PR DESCRIPTION
**Pull request by NickM-27**

### CODE REVIEW

File: docs/docs/configuration/index.md [hunk 1]
```diff
@@ -477,6 +477,18 @@ cameras:
       order: 0
       # Optional: Whether or not to show the camera in the Frigate UI (default: shown below)
       dashboard: True
+      # Optional: Override browser locale and show time in 12 / 24 hour format (default: use browser locale)
+      use12hour: False
```
Comment (blakeblackshear) [inline docs/docs/configuration/index.md:481]: Looking at this again, I think it makes more sense to use an enum here called `time_format` with options `12-hour`, `24-hour`, `browser`.

File: docs/docs/configuration/index.md [hunk 2]
```diff
@@ -477,6 +477,18 @@ cameras:
...
+      # Optional: Set the date style for a specified length. 
+      # Options are: full, long, medium, sort (default: shown below).
+      date_style: short
```
Comment (blakeblackshear) [inline docs/docs/configuration/index.md:484]: Nobody is going to know what full, long, medium, short mean. We should include an example of each.

**Review by blakeblackshear — CHANGES_REQUESTED** [review]

File: docs/docs/configuration/index.md [hunk 3]
```diff
@@ -486,6 +486,29 @@ ui:
   timezone: None
   # Optional: Use an experimental recordings / camera view UI (default: shown below)
   experimental_ui: False
+  # Optional: Set the time format used.
+  # Options are browser, hour12, or hour24 (default: shown below)
+  time-format: browser
```
Comment (blakeblackshear) [inline docs/docs/configuration/index.md:491]: I think this should be an underscore, not a dash

Comment (blakeblackshear) [reply docs/docs/configuration/index.md:491]: Also, the options are `hours12` and `hours24`, not `hour12`, etc

File: web/src/utils/dateUtil.ts [hunk 4]
```diff
@@ -37,19 +37,20 @@ export const getNowYesterdayInLong = (): number => {
  */
 interface DateTimeStyle {
   timezone: string;
-  use12hour: boolean | undefined;
+  time_format: 'browser' | '12hour' | '24hour';
```
Comment (blakeblackshear) [inline web/src/utils/dateUtil.ts:40]: The frontend is using `hour` and the backend is using `hours`

Comment (NickM-27) [reply web/src/utils/dateUtil.ts:40]: Geez my bad, do you have a preference for which one to use?

Comment (blakeblackshear) [reply web/src/utils/dateUtil.ts:40]: Just did some quick googling, and I think `12hour` and `24hour` seem to be most common way to refer to time formats.

Comment (NickM-27) [reply web/src/utils/dateUtil.ts:40]: Do you know a good way to do that? The enum variable name can't start with a digit, not sure if there is a workaround for that

Comment (NickM-27) [reply web/src/utils/dateUtil.ts:40]: Nevermind looks like it works on both sides with just the string value

**Review by blakeblackshear — APPROVED** [review]

**Comment by herostrat** [comment]
Just thinking, should we use the new time/date style for the timestamp_style of the in-feed timestamp?

**Pull request merged**